### ISSUE TITLE
Inconsistent Behavior in Type Checking for `collections.abc.Callable` and `collections.abc`

### ISSUE THREAD

**Issue by brearleyt** [microsoft/pyright#6527]
**Describe the bug**
Pyright exhibits inconsistent behaviour when checking types for `Callable` imported from `collections.abc` directly versus from `collections` module. 

**Code or Screenshots**
This code passes the type checker as expected:

```python
from collections.abc import Callable


def f() -> None:
    ...


assert isinstance(f, Callable)
```

However, this code causes pyright to raise an error:

```py
from collections import abc


def f() -> None:
    ...


assert isinstance(f, abc.Callable)
```

Output:
```
example.py:8:22 - error: Type of "Callable" is partially unknown
  Type of "Callable" is "type[(...) -> Unknown]" (reportUnknownMemberType)
```

Parameterised generics cannot be used with instance and class checks, so my assumption is we shouldn't be raising this error in this case. 

**VS Code extension or command-line**
pyright 1.1.337 via the command-line.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.338, which I just published. It will also be included in a future release of pylance.
